### ISSUE TITLE
Reduction server permission to us-docker.pkg.dev/vertex-ai-restricted/training/reductionserver:latest

### ISSUE THREAD

**Issue by almarengo** [GoogleCloudPlatform/vertex-ai-samples#2176]
I'm trying the Distributed Training with Reduction Server but when I'm trying to use the reduction server image `us-docker.pkg.dev/vertex-ai-restricted/training/reductionserver:latest` I get the error the SA doesn't have permission to access the image. 

My SA has owner role. When i look the image up I get the same permission issue
![image](https://github.com/GoogleCloudPlatform/vertex-ai-samples/assets/77657772/f7b959fa-114f-4c20-9134-655bc81f760a)

Am I doing something wrong?

**Comment by gericdong** [comment]
@almarengo: is this still an issue for you?

**Comment by almarengo** [comment]
@gericdong yes, I still can’t find the image

**Comment by gericdong** [comment]
@almarengo I will help you with that. Are you using a custom service account to pull images? According to the docs here https://cloud.google.com/vertex-ai/docs/general/custom-service-account#configure, you can't configure a custom service account to pull images from Artifact Registry. Vertex AI uses the default service account to pull images.
Can you please try using the the default service account and see if this is the case? Thanks.

**Comment by almarengo** [comment]
Thanks @gericdong for your help. I’m using the default service account on the vertex ai instance that launches the training job. I’m not passing any custom SA when I run the job so I’m assuming it’s using the same SA to run the job…

**Comment by gericdong** [comment]
@almarengo no problem. Would it be possible for you to provide me the code that you use to pull and use the image? I can try it myself to see if we can diagnose the issue.

**Comment by almarengo** [comment]
Sure @gericdong I can share my code. It will have to be in a couple of weeks as I’m not around my workstation. I’m not sure it it makes any difference , but I can’t seem to be able to see the image in the GCP online repository (like I’m able to do with any other image from training to running model endpoints, etc..) I get the permission issue I posted above. Can you see the repo?

**Comment by gericdong** [comment]
@almarengo yes I can see the repo. And I am not able to reproduce your issue in my environment. It'd help if you can share your code and procedure. Meanwhile, I will try setting up a different environment to see if I get the permission issue. Thanks.

**Comment by almarengo** [comment]
Sure @gericdong, I’ll share the code as soon as possible. Still, I think the error has something to do with me not even being able to see the repo. The fact that the repo is in a ‘restricted’ folder makes me think I need some super-user access. I’ll get back to you asap. Thanks so much for your help!

**Comment by gericdong** [comment]
Thanks @almarengo. We are also looking into it to see if there is a configuration issue with the repo.

**Comment by gericdong** [comment]
@almarengo can you please provide your project ID and job ID if possible? Our product team is helping to look into the issue.

**Comment by almarengo** [comment]
@gericdong I cannot find the job ID because the job does not start. I get this error right before the job is even launched

`google.api_core.exceptions.FailedPrecondition: 400 Vertex AI Service Agent service-256068731062@gcp-sa-aiplatform-cc.iam.gserviceaccount.com does not have permission to access Artifact Registry repository projects/vertex-ai-restricted/locations/us/repositories/training.`

**Comment by gericdong** [comment]
@almarengo thank you. We will be looking into it.

**Comment by andrewferlitsch** [comment]
product team is currently working on this issue

**Comment by gericdong** [comment]
@almarengo the team tried but unfortunately wasn't able to reproduce the issue. Not sure what we can do from here.

**Comment by gericdong** [comment]
Closed due to inactivity.
